Dockerfile

```diff
@@ -0,0 +1,25 @@
+# kovri - I2P but in CPP
+# Copyright (c) 2015-2017, The Kovri I2P Router Project (see LICENSE.md)
+# docker build -t geti2p/kovri .
+# KOVRI_PORT=42085 && sudo docker run -p $KOVRI_PORT --env KOVRI_PORT=$KOVRI_PORT geti2p/kovri
+
+FROM alpine:3.5
+RUN apk add --update --no-cache -t .kovri-dev g++ make cmake binutils boost-dev libressl-dev
+ENV KOVRI_HOST 0.0.0.0
+ENV KOVRI_PORT 24085
+ADD . kovri/
```

Comment:
This[1] is the best resource I could find on strategies on getting code in to Docker containers. I'm going to add some stuff to the .dockerignore and leave it as is for now. Mainly because I know it works.

[1]: https://ryanfb.github.io/etc/2015/07/29/git_strategies_for_docker.html